std/experimental/ndslice/package.d

```diff
@@ -57,6 +57,12 @@ Multidimensional view is presented by $(SUBREF slice, Slice) type.
 ------
 auto matrix = new double[12].sliced(3, 4);
```

Comment:
1. I want this example be in the beginning
2. DDOC unittest do not work with module declarations
3. the same unittest already exists without DDOC